app/src/components/postit/PostIt.tsx

```diff
@@ -0,0 +1,77 @@
+import { css, cx } from '@emotion/css';
+import { color, font } from '../../styles/index';
+import { Props } from '../button';
+
+function PostIt(props: Props) {
+  const commonStyle = css`
+    ${font.weight.regular}
+    background-color: white;
+    box-shadow: 0px 0px 5px 0 rgb(193, 193, 197);
+    padding: 16px 35px 16px 16px;
+    outline: none;
+  `;
```

Comment:
```suggestion
function PostIt(props: Props) {
  const { children, small } = props;

  const commonStyle = css`
    ${font.weight.regular}
    background-color: white;
    box-shadow: 0px 0px 5px 0 rgb(193, 193, 197);
    padding: 16px 35px 16px 16px;
    outline: none;
  `;
```

props.children이랑 이 방법이랑 성능이나 기능면에서 큰 차이는 없는데 더 간결하고 확장성이 좋습니다.

밑에 있는 다른 css 변수랑 겹쳐서 그런거면 변수 네이밍을 다시 생각해보는게 좋을듯 해요

Reply:
넵 알겠습니다, 감사합니다 ^__^